### DISCUSSION TITLE
Keywords aren't improving results in many cases

### DISCUSSION THREAD

**Discussion by orozCoding** [deepgram/community#155] [General help]
We're trying to optimize the results in our transcripts, and keywords aren't helping in many cases.

For example, we'd like to see 'Ali Ryder' instead of 'Ali Rider'

Passing keywords Ali and Rider (tried different intensities) aren't improving the result.

Similar scenario: We want 'Karin Nilsson', but we keep getting 'Caroline Nielsen'

I'll leave some data from an example POST request
```
POST /v1/listen?utterances=true&punctuate=true&paragraphs=true&detect_topics=true&summarize=true&smart_format=true&measurements=true&tier=enhanced&redact=pci&redact=ssn&model=general&keywords=Karin%3A3&keywords=Nilsson%3A3&keywords=rider%3A-10
````

<img width="470" alt="image" src="https://github.com/deepgram/community/assets/92640684/8d22cbe4-d2b0-40a8-9a68-9e0128569c69">

We're using `model=general`.

**Comment by jjmaldonis** [comment]
Hey @orozCoding thanks for the report, we're looking into this. We have a few things we'd like to dig into and some additional info from you would be very helpful.

1. If you use `tier=base` rather than `tier=enhanced` (while keeping `model=general`), does it improve the results?
2. Can you copy-paste the relevant output for the names/keywords that are not working correctly? We are specifically interested in the `confidence` for those words.
